travis-ci/vmtest/run_selftests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -eux
+
+cd libbpf/selftests/bpf
+
+echo TEST_PROGS
+./test_progs ${BLACKLIST:+-b$BLACKLIST} ${WHITELIST:+-t$WHITELIST}
```

Comment:
It will be slightly more then that, BLACKLIST and WHITELIST should be copied to disk image mount along with tests to be available in vmtest.
Will do.

Reply:
yeah, that copying problem is annoying... We should look at just mounting it through 9P filesystem (that's what I do locally when working with qemu)